### ISSUE TITLE
Dirt Rally will not load with Revive

### ISSUE THREAD

**Issue by Raging-Beard** [LibreVR/Revive#209]
Hi, Rally loaded the first time great on my Vive, but since then only a window flashes up too quick to read then nothing happens. I have tried reinstalling Rally and tried different Injector versions.

Any help or ideas would be much appreciated. Thank you.

**Comment by ccsled** [comment]
Try the patch in this thread. Worked for me(ReviveDirt.zip) . https://github.com/LibreVR/Revive/issues/201

**Comment by LibreVR** [comment]
First, make sure you are running the latest version. Then post your log files from `%APPDATA%\Revive` here.

**Comment by Raging-Beard** [comment]
Hi meant to reply some time ago just to say it worked using the specific patch made I found on another thread, which I believe is the same one linked above. 

Many thanks, it's even better just clicking play from Steam instead of dragging the exe.

**Comment by supermanscottbristol** [comment]
I cannot get Dirt Rally to launch with similar issues to above.
I get a flash of black on screen, nothing in the vive headset.
I'm running Oracle Home and Steam VR.

I've attached 2 log files, one from using the standard injector, and one from using the specific patch from the other thread. 

Tried with x86 and x64

Any ideas?

Thanks
[revivedirt.txt](https://github.com/LibreVR/Revive/files/403539/revivedirt.txt)
